.eslintrc

```diff
@@ -1,90 +1,9 @@
 {
-  "env": {
-    "node": true,
-    "browser": true
-  },
-  "rules": {
-    "camelcase": 0,
-    "no-comma-dangle": 2,
-    "new-cap": 0,
-    "no-process-exit": 0,
-    "max-len": [
-      1,
-      110,
-      2
-    ],
-    "no-new-require": 2,
-    "space-in-brackets": 0,
-    "wrap-iife": [
-      2,
-      "outside"
-    ],
-    "no-self-compare": 2,
-    "no-unused-expressions": 0,
-    "no-warning-comments": 2,
-    "max-params": [
-      1,
-      4
-    ],
-    "quotes": [
-      2,
-      "single",
-      "avoid-escape"
-    ],
-    "no-else-return": 2,
-    "max-depth": 2,
-    "max-nested-callbacks": [
-      2,
-      5
-    ],
-    "block-scoped-var": 2,
-    "no-lonely-if": 2,
-    "handle-callback-err": 2,
-    "no-extra-parens": 2,
-    "no-nested-ternary": 2,
-    "brace-style": [
-      2,
-      "1tbs"
-    ],
-    "no-path-concat": 1,
-    "no-eq-null": 2,
-    "no-use-before-define": [
-      2,
-      "nofunc"
-    ],
-    "func-style": [
-      2,
-      "declaration"
-    ],
-    "semi": [
-      2,
-      "always"
-    ],
-    "space-unary-ops": 2,
-    "valid-jsdoc": [
-      2,
-      {
-        "prefer": {
-          "return": "returns"
-        }
-      }
-    ],
-    "strict": 0,
-    "complexity": [0, 10],
-    "no-mixed-requires": [
-      2,
-      true
-    ],
-    "no-underscore-dangle": 0,
-    "space-after-keywords": [
-      2,
-      "always"
-    ],
-    "curly": 0
-  },
-  "space-in-parens": [
-    2,
-    "never"
+  "extends": [
+    "eslint-config-airbnb",
+    "eslint-config-algolia"
   ],
-  "globals": {}
+  "rules": {
+    "strict": 0
```

Comment:
I am not sure it brings me anything since eslint already detect unused variables and undeclared global variables.

So since it does not brings me anything but one more line per file, I prefer not to use it. Also 'use strict'; will at some point turn into 'use really-strict' etc..

I think we already had the discussion at some point early on.